release/release.go

```diff
@@ -303,6 +346,26 @@ func genRKE2ReleaseNotes(tmpl *template.Template, milestone string, rd rke2Relea
 	return b, nil
 }
 
+func genUIReleaseNotes(tmpl *template.Template, _ string, rd uiReleaseNoteData) (*bytes.Buffer, error) {
+	uiTemplate := fmt.Sprintf(dashboardReleaseNoteTemplate, "ui")
+	tmpl = template.Must(tmpl.Parse(uiTemplate))
+	b := bytes.NewBuffer(nil)
+	if err := tmpl.ExecuteTemplate(b, uiRepo, rd); err != nil {
+		return nil, err
+	}
+	return b, nil
+}
+
+func genDashboardReleaseNotes(tmpl *template.Template, _ string, rd dashboardReleaseNoteData) (*bytes.Buffer, error) {
+	dashboardTemplate := fmt.Sprintf(dashboardReleaseNoteTemplate, "dashboard")
+	tmpl = template.Must(tmpl.Parse(dashboardTemplate))
+	b := bytes.NewBuffer(nil)
+	if err := tmpl.ExecuteTemplate(b, dashboardRepo, rd); err != nil {
+		return nil, err
+	}
+	return b, nil
+}
+
```

Comment:
Not saying that we should do it, but I think this is a case where we could use the new go generics